src/artifacts-helper/codespaces_artifacts_helper_keyring/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[project]
+name = "codespaces-artifacts-helper-keyring"
+version = "0.1.0"
+description = "Keyring backend to retrieve credentials for Azure Artifacts on Codespaces using https://github.com/microsoft/ado-codespaces-auth"
+authors = [{ name = "Delilah Wu", email = "delilahwu@microsoft.com" }]
```

Comment:
Package metadata needs updating before we publish. E.g. change author to Microsoft Coporation